src/client/components/componentWithSliders/allSliders/blockOfSliders/slider/slider.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Slider from 'react-rangeslider';
+import 'react-rangeslider/lib/index.css';
+
+import './slider.css';
+
+class OneSlider extends Component {
+  constructor (props, context) {
+    super(props, context);
+    this.state = {
+      sliderValue: props.value,
+      maxValue: 1,
+      minValue: 0,
+      step: 0.01
+    };
+
+  }
+
+  handleChange = (value) => {
+      value = parseFloat(value.toFixed(2));
```

Comment:
It’s better to cal variables more informative, for eg here what value do you have, sliderValue

Reply:
done